src/hooks/useKeyDown.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from 'react'
+
+type Bindings = Record<string, () => void>
+
+export default (bindings: Bindings) => {
+  useEffect(() => {
+    const onKeyDown = (e: KeyboardEvent) => {
```

Comment:
This kind of hook will be useful 👍 Could you add a check that input elements are ignored or pass a ref to control from which element the keypresses are accepted, eg. https://github.com/streamr-dev/streamr-platform/blob/0539dc725fb86928d442ca2207541944d677fc7c/app/src/shared/utils/isEditableElement.js#L5

Now if you type `0` in the search it will reset the map.

Reply:
Good point 👍